libmozdata/bugzilla.py

```diff
@@ -453,20 +455,46 @@ def __get_bugs_by_search(self):
         """
         url = Bugzilla.API_URL + '?'
         header = self.get_header()
+        specials = {'count_only', 'limit', 'order', 'offset'}
         for query in self.bugids:
             if isinstance(query, six.string_types):
                 url = Bugzilla.API_URL + '?' + query
-                params = None
-            else:
+                self.bugs_results.append(self.session.get(url,
+                                                          headers=header,
+                                                          verify=True,
+                                                          timeout=self.TIMEOUT,
+                                                          background_callback=self.__bugs_cb))
+            elif not (specials & set(query.keys())):
```

Comment:
Nit: You could use `isdisjoint` which is a bit more readable. `specials.isdisjoint(query)` or more readable `specials.isdisjoint(query.keys())`